bravado/swagger_model.py

```diff
@@ -5,6 +5,10 @@
 import os.path
 
 import yaml
+try:
```

Comment:
nit. I'm wondering if it would be better to move this into `compact.py` (as we do for `json` vs `simplejson`)
Anyway this is not blocking.